### ISSUE TITLE
Button within focus trap that uses Clipboard js does not work

### ISSUE THREAD

**Issue by asaadmahmood** [focus-trap/focus-trap#91]
A button that uses clipboard js does not work if it exists within the focustrap.
https://clipboardjs.com/

**Comment by Tckool** [comment]
Sorry not sure if you've found the answer to this already but I ran into the same issue recently. Seems that if you use clipboard js with `data-clipboard-text`, then what happens is that clipboard js will create a temporary element on the page and copy the value of it. 

The issue is that because the element created is outside of the focus trap, it does not allow it to be focused and cannot copy the value. 

So my workaround was to use `data-clipboard-target` and choose an element inside of my focus trap

**Comment by stefcameron** [comment]
Closing issues older than 2020. Please LMK if this is still a problem.
